R/te_data.R

```diff
@@ -77,26 +79,49 @@ setValidity(
   }
 )
 
+# Show
+setMethod(
+  "show",
+  c(object = "te_outcome_data"),
+  function(object) {
+    if (!length(object@data)) {
+      catn("No outcome data, use load_expanded_data()")
+    } else {
+      catn("Outcome data")
+      catn("N:", object@n_rows, "observations from", object@n_ids, "patients in", length(object@periods), "trial periods")
+      catn("Periods:", object@periods)
+      if (length(object@subset_condition)) catn("Subset condition:", object@subset_condition)
+      if (length(object@p_control)) catn("P-control:", object@p_control)
```

Comment:
Maybe we explain this a bit more
```suggestion
      if (length(object@p_control)) catn("Sampling control observations with probability:", object@p_control)
```